### PR TITLE
Cave

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Our first sample project is working in the CAVE! I "hard coded" the changes in Unity directly so they don't show up in code yet but they're all present in the  `sample.unity` scene

- Update README
- Add `CAVE.md` for documenting steps for running w3d in the cave
- Add `HMD.md` for documenting steps for running w3d in a headset
- Rename `UNITY.md` docs as `Writing3D.md`
- Add partial script for building a scene
  - Leaving commented out for now as they're not completed
- A couple small bug fixes to `w3d_translator.py` script

**CAVE NOTES**
- Added MiddleVR package (local tarball)
- Added a `Template Camera` - this is the prefab applied to the stereoscopic cameras that MiddleVR creates
- `MVRInteractable.cs` extends MiddleVR's `MVRActor` script to handle events/actions
  - `MVRActor` adds an event which supplies if a button is pressed and an int for the given button
  - We invoke the corresponding XRI event in `LinkManager` based on `button` and `isPressed`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,55 +4,37 @@ Translate Writing3D xml projects into Unity projects. Each scene file is automat
 
 ## Install
 
-```console
+```sh
```
Comment (mirestrepo) [inline README.md:7]: Who are these instructions for? If only developers, then install via `poetry` is okay, but If we think that anyone could install the package, then ideally they could instead do 
`pipx install git+https://github.com/brown-ccv/w3d-translator`

and  futhermore w3d_translator.py could be configured as a console script so one just needs to do something along lines

```
w3d_translate indir outdir [OPTIONS]
```

This doesn't need to be addressed now, but I often find helpful to separate the normal user instructions from the development instructions

File: docs/CAVE.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Unity Setup for CAVE Deployment
+
+## Writing3D Notes
+
+- Root object needs to move to be the first in the hierarchy
+- Add MVR Interactable script whenever I add a LinkManager
```
Comment (mirestrepo) [inline docs/CAVE.md:6]: Is MVR MiddleVR? Maybe define that somewhere?

**Review by mirestrepo — APPROVED** [review]
Woohoo! It's awesome that first projects is there!

It would be nice to at some point have @kmilo9999 or maybe even someone else in the group to run through instructions in the CAVE if that makes sense? I always find that instructions work best when at least 2 people walk. through them.

Awesome to see this coming along!

**Comment by kmilo9999** [comment]
Sure, we can test it in VR office hours. Happy to help !

**Review by kmilo9999 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RobertGemmaJr) [reply README.md:7]: They're for developers, which is really just us?

I'm curious who may end up running the script? Is that something we expect John Caley to do at some point or are we going to translate all of the projects?

File: docs/CAVE.md (hunk 2 above)
Comment (RobertGemmaJr) [reply docs/CAVE.md:6]: Yeah, most of their scripts started with `MVR`. I can add a note at the top of the doc

**Pull request merged**